demos/background_subtraction_demo/cpp_gapi/main.cpp

```diff
@@ -82,6 +84,41 @@ static cv::gapi::GKernelPackage getKernelPackage(const std::string& type) {
     GAPI_Assert(false && "Unreachable code!");
 }
 
+cv::gapi::wip::onevpl::CfgParam createFromString(const std::string &line) {
+    using namespace cv::gapi::wip;
+
+    if (line.empty()) {
+        throw std::runtime_error("Cannot parse CfgParam from emply line");
+    }
+
+    std::string::size_type name_endline_pos = line.find(':');
+    if (name_endline_pos == std::string::npos) {
+        throw std::runtime_error("Cannot parse CfgParam from: " + line +
+                                 "\nExpected separator \":\"");
+    }
+
+    std::string name = line.substr(0, name_endline_pos);
+    std::string value = line.substr(name_endline_pos + 1);
+
+    return cv::gapi::wip::onevpl::CfgParam::create(name, value,
+                                                   /* vpp params strongly optional */
+                                                   name.find("vpp.") == std::string::npos);
+}
+
+static std::vector<cv::gapi::wip::onevpl::CfgParam> parseVPLParams(const std::string& cfg_params) {
+    std::vector<cv::gapi::wip::onevpl::CfgParam> source_cfgs;
+    std::stringstream params_list(cfg_params);
+    std::string line;
+    while (std::getline(params_list, line, ';')) {
+        source_cfgs.push_back(createFromString(line));
+    }
+    return source_cfgs;
+}
```

Comment:
I believe user apps shouldn't reinvent this bycicle every time.. We should ship some default implementation for this, please add it to the todo list :)

Reply:
From my opinion it should be default args-list parsing routine (with find-first-one-separator string ",;\n\r") which provides a `std::list<std::string>` and then transforming this list into cfgParams by using `createFromString` in loop on them

Reply:
Let it be as part of this issue: https://github.com/opencv/opencv/issues/22506